sql/functions.sql

```diff
@@ -0,0 +1,23 @@
+--liquibase formatted sql
+
+--changeset emokoena:create-count-unsold-kegs-from-batch
+CREATE FUNCTION CountUnsoldKegsFromBatch(batch_id INT)
+RETURNS INT
```

Comment:
Use integer (lower case) for variable types

Reply:
Cool On it